### PR TITLE
fMRIVolume Related Pull Request — Supporting processing w/o T2w image

### PR DESCRIPTION
**Pull request by sivcek**
Adds the option to run fMRIVolume even w/o T2w image present. It uses T1w image for reference instead.

### CODE REVIEW

File: fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh [hunk 1]
```diff
@@ -409,15 +409,23 @@ case $DistortionCorrection in
         Files="PhaseOne_gdc_dc PhaseTwo_gdc_dc SBRef_dc"
         for File in ${Files}
         do
+            # check if T2w image exists
+            if [ `${SubjectFolder}/T1w/T2w_acpc_dc.nii.gz` -eq 0 ]; then
```
Comment (coalsont) [inline fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh]: Backticks do command substitution, and the image file, whether or not it exists, is not an executable.  You want double quotes, `""`.

Also, `-eq 0` is not how you test for file existence (it should error, as the string does not parse as an integer).  You want `-f <file>`.

Comment (coalsont) [reply fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh]: Oh, I see - you forgot to put `imgtest` in the backticks.

Comment (sivcek) [reply fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh]: Yes. I changed the code we used to have to follow @mharms guidance and the imtest fell out during copy/paste. Corrected now.

**Comment by sivcek** [comment]
Simplified to use T1w image as reference.

File: fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh [hunk 2]
```diff
@@ -409,15 +409,23 @@ case $DistortionCorrection in
         Files="PhaseOne_gdc_dc PhaseTwo_gdc_dc SBRef_dc"
         for File in ${Files}
         do
+            # check if T2w image exists
+            if [ `${FSLDIR}/bin/imtest ${SubjectFolder}/T1w/T2w_acpc_dc.nii.gz` -eq 0 ]; then
+                ReferenceImage=${SubjectFolder}/T1w/T2w_acpc_dc.nii.gz
+            else
+                ReferenceImage=${SubjectFolder}/T1w/T1w_acpc_dc.nii.gz
+            fi
```
[the remark this answers is not in the document]
Comment (sivcek) [reply fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh]: Done that.

File: fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh [hunk 3]
```diff
@@ -409,15 +409,17 @@ case $DistortionCorrection in
         Files="PhaseOne_gdc_dc PhaseTwo_gdc_dc SBRef_dc"
         for File in ${Files}
         do
+            ReferenceImage=${SubjectFolder}/T1w/T1w_acpc_dc.nii.gz
```
Comment (mharms) [inline fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh]: `ReferenceImage` is independent of the `for File in ${Files}` loop, so can be defined above that.

Comment (sivcek) [reply fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh]: Moved outside of the loop.

**Comment by sivcek** [comment]
Joined with other fMRIVolume PRs into PR <https://github.com/Washington-University/HCPpipelines/pull/156>

**Pull request closed without merging**